src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -1089,6 +1089,10 @@ private RuntimeType[] PopulateInterfaces(Filter filter)
                     return list.ToArray();
                 }
 
+                [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:UnrecognizedReflectionPattern",
+                    Justification = "Calls to ResolveTypeHandle technically require all types to be kept" +
```

Comment:
Consider adding spaces to the end of these strings? (The justification text is broken into 3 parts, concatenated with `+`; these parts, when concatenated, end up with `keptBut` and `thatshould`)